install/ui/src/freeipa/dns.js

```diff
@@ -1226,7 +1226,21 @@ IPA.dns.get_record_metadata = function() {
             ],
             adder_attributes: [],
             columns: ['txt_part_data']
-        }
+        },
+        {
+            name: 'urirecord',
+            attributes: [
+                'uri_part_priority',
```

Comment:
Thinking about the 'doc' text from API,
```javascript
'uri_part_priority',
```
can be replaced with standard field definition, e.g., with:
```javascript
{
    name: 'uri_part_priority', 
    tooltip: '@mo-param:dnsrecord:uri_part_priority:doc'
}
```